src/component/tooltip/TooltipView.ts

```diff
@@ -628,8 +628,9 @@ class TooltipView extends ComponentView {
             this._tooltipModel
         ]);
 
+        // fix 10537
         const tooltipTrigger = tooltipModel.get('trigger');
-        if (tooltipTrigger != null && tooltipTrigger !== 'item') {
+        if (!tooltipTrigger || tooltipTrigger === 'none') {
             return;
         }
```

Comment:
This change only makes tooltip trigger on the item even if it's configured as `axis`. It will break other series and can't get the expected result of radar. In radar, the expected result is to show all the values of one indicator if tooltip.trigger is `axis`

Reply:
> I think this change only makes tooltip trigger on the item even if it's configured as `axis`. It will break other series and can't get the expected result of radar. In radar, the expected result is to show all the values of one indicator if tooltip.trigger is `axis`

got it